shuffle-plugin/pom.xml

```diff
@@ -61,7 +61,7 @@
         <dependency>
           <groupId>org.openucx</groupId>
           <artifactId>jucx</artifactId>
-          <version>1.9.0</version>
+          <version>1.11.0</version>
```

Comment:
noticed the snapshot is no longer being used, is this the version with the fixes needed already?

Reply:
Yes this version is of master Apr 12. It has everything JUCX needed for now